lib/dotcom/alerts/disruptions/subway.ex

```diff
@@ -0,0 +1,55 @@
+defmodule Dotcom.Alerts.Disruptions.Subway do
+  @moduledoc """
+  Disruptions are alerts that have `service_impacting_effects` grouped by `service_range`.
+  """
+
+  import Dotcom.Alerts, only: [service_impacting_alert?: 1]
+  import Dotcom.Routes, only: [subway_route_ids: 0]
+  import Dotcom.Utils.ServiceDateTime, only: [service_range: 1]
+
+  alias Alerts.Alert
+  alias Dotcom.Utils
+
+  @alerts_repo Application.compile_env!(:dotcom, :repo_modules)[:alerts]
+
+  @doc """
+  Disruptions that occur any time after today's service range.
+  """
+  @spec future_disruptions() :: %{Utils.ServiceDateTime.named_service_range() => [Alert.t()]}
+  def future_disruptions() do
+    disruption_groups() |> Map.take([:later_this_week, :next_week, :after_next_week])
+  end
+
+  @doc """
+  Disruptions that occur during today's service range.
+  """
+  @spec todays_disruptions() :: %{Utils.ServiceDateTime.named_service_range() => [Alert.t()]}
+  def todays_disruptions() do
```

Comment:
I may be reading this wrong, but it looks like this will include alerts from earlier today, even if their effect periods have ended. If so, we should double-check the requirements, because I had thought that we only want to include alerts if they're either currently active or will be active later in the day.

Either way, I think it would be good to have a test explicitly highlighting the "earlier-today" alerts behavior.

Reply:
This is grabbing todays disruptions. If the view doesn't want ones that are earlier than `now` they can remove them. The intention here isn't to do all of the work for the only current consumer. They will still have to do grouping and sorting by line if they want it.

Reply:
And, it will only get those alerts if the effect period ended after the start of the service day. So, the disruption effected 'today'.